ansible/roles/pis/files/pi-heartbeat.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Pi Heartbeat
+After=systemd-modules-load.service

```

Comment:
It may be necessary to do an `After=network-online.target` to ensure the network is up before the heartbeater runs.